project/src/main/webapp/details-page/details.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="theme-color" content="#0000000">
+    <title>Case Details</title>
+    <link rel="stylesheet" href="details_style.css">
+    <script src="details_script.js"></script>
+</head>
+<body>
+<div id="content">
+    <form id="header" action="../index.html">
+        <input type="submit" value="Back"/>
+    </form>
+    <div style="width: 100%; overflow: hidden;">
```

Comment:
It'd probably make sense to create classes or IDs for this div and the one it immediately surrounds.